diff --git a/node_modules/@svgdotjs/svg.js/svg.js.d.ts b/node_modules/@svgdotjs/svg.js/svg.js.d.ts
index 52d8bf8..4ec68dc 100644
--- a/node_modules/@svgdotjs/svg.js/svg.js.d.ts
+++ b/node_modules/@svgdotjs/svg.js/svg.js.d.ts
@@ -377,7 +377,7 @@ declare module "@svgdotjs/svg.js" {
         constructor(array?: ArrayAlias<ArrayXY> | number[]);
 
         toLine(): LineAttr;
-        at(pos: number): PointArray;
+        at(pos: number): ArrayXY;
         transform(m: Matrix | MatrixLike): PointArray
         move(x: number, y: number): this;
         size(width: number, height: number): this;
@@ -436,7 +436,7 @@ declare module "@svgdotjs/svg.js" {
         size(width: number, height: number): this;
         equalCommands(other: PathArray): boolean
         morph(pa: PathArray): this
-        at(pos: number): PathArray
+        at(pos: number): PathCommand
         parse(array?: ArrayAlias<PathCommand> | PathArrayAlias): PathCommand[];
         bbox(): Box;
         to(a: any): Morphable
